responses/__init__.py

```diff
@@ -570,6 +570,8 @@ def __init__(
         self.response_callback = response_callback
         self.passthru_prefixes = tuple(passthru_prefixes)
         self.target = target
+        self._patcher = None
+        self._matches = []
```

Comment:
Doesn't the reset() call on line 568 do this?

Reply:
@markstory 
yes, it does, but by python convention all class attributes should be initialized in `__init__` method

Reply:
Ok. Couldn't we remove the call to `reset()` then?

Reply:
@markstory 
`reset` also resets calls:
`self._calls.reset()`